### PR TITLE
Use papermill to generate BM3

### PR DESCRIPTION
**Pull request by wd15**
Use papermill to generate BM3 results page. In addition:

 - use requests-cache to cache downloaded data to improve notebook execution time
 - remove uploads from jah5759 as the account is dead
 - Fix a few broken meta.yaml's
 - Fix some broken links
 - Redirect 3a.1 to new notebook version of BM3 results

Todo:

 - [x] cache the requests_cache file for github actions
 - [x] rebase

### CODE REVIEW

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by wd15** [comment]
@tkphd, ok, this if finally ready for you to review. Thanks!

File: _data/simulation_list.yaml [hunk 1]
```diff
@@ -73,15 +73,15 @@
 - simulations/fenics_1c_ivan/meta.yaml
 - simulations/prismspf_6b/meta.yaml
 - simulations/inl_moose_6a/meta.yaml
-- simulations/moose_2b_jah/meta.yaml
-- simulations/moose_2a_jah/meta.yaml
-- simulations/moose_2d_jah/meta.yaml
-- simulations/moose_1a_jah/meta.yaml
-- simulations/moose_1b_jah/meta.yaml
-- simulations/moose_2c_jah/meta.yaml
+# - simulations/moose_2b_jah/meta.yaml
```
Comment (tkphd) [inline _data/simulation_list.yaml:76]: Why are these excluded? Is it more appropriate to comment them out, or cut the lines?

Comment (wd15) [reply _data/simulation_list.yaml:76]: The data for those results is stored in [jah5759](https://github.com/jah5759)'s GitHub account. The account seems to have disappeared. Another good reason to switch to a DOI service.

I think it's best to comment as it kind of remind me to investigate it later or at least that those results have been skipped.

Comment (wd15) [reply _data/simulation_list.yaml:76]: @jah5759

File: benchmarks/benchmark4.ipynb [hunk 2]
```diff
@@ -134,7 +134,7 @@
    "source": [
```
Comment (tkphd) [inline benchmarks/benchmark4.ipynb:134]: <p>The equation failing to render needs escapes on the <code>{</code>,<code>}</code> delimiters:</p><p> </p><pre class="ql-syntax" spellcheck="false">\begin{equation}
\frac{\partial\eta}{\partial t} = \nabla\cdot\left[M\nabla \left\{ \frac{\delta \mathcal{F}}{\delta\eta} \right\} \right],
\end{equation}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Review by tkphd — CHANGES_REQUESTED** [review]
Looks good overall. Some minor adjustments would help.

File: benchmarks/benchmark4.ipynb (hunk 2 above)
Comment (wd15) [reply benchmarks/benchmark4.ipynb:134]: It isn't failing to render for me or on the [website](https://random-cat-1485.surge.sh/benchmarks/benchmark4.ipynb/). Where are you seeing this?

Comment (tkphd) [reply benchmarks/benchmark4.ipynb:134]: Ah, you're right! It doesn't render on ReviewNB, but it looks fine on the website. Must be different math renderers.

**Comment by wd15** [comment]
> View / edit / reply to [this conversation](https://app.reviewnb.com/usnistgov/pfhub/pull/1485/discussion/#comment-1442170945) on ReviewNB
> 
> _tkphd commented on 2023-02-23T17:33:33Z_ ----------------------------------------------------------------
> 
> Same complaint about listing the axes as the chart title as the previous PR.

Ok, that's the template. I haven't actually re-run the template and it's just used to generate the actual benchmark notebook. See [this](https://random-cat-1485.surge.sh/results/benchmark3a.1.ipynb/) to see how BM3 actually looks.

**Comment by wd15** [comment]
<p>Ok, that's the template. I haven't actually re-run the template and it's just used to generate the actual benchmark notebook. See [this](https://random-cat-1485.surge.sh/results/benchmark3a.1.ipynb/) to see how BM3 actually looks.</p>

---

 <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1485/discussion/#comment-1442170945'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by wd15** [comment]
<p>Again, that's the template, see here for the <a href="https://random-cat-1485.surge.sh/results/benchmark3a.1.ipynb/" target="_blank">rendered template</a>.</p>

---

 <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1485/discussion/#comment-1442170973'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: _data/simulation_list.yaml (hunk 1 above)
Comment (wd15) [reply _data/simulation_list.yaml]: Now purged

**Comment by wd15** [comment]
@tkphd, concerns addressed. Please look again. Thanks.

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

File: benchmarks/benchmark4.ipynb (hunk 2 above)
Comment (wd15) [reply benchmarks/benchmark4.ipynb:134]: Oh, it's failing on ReviewNB only.

Comment (wd15) [reply benchmarks/benchmark4.ipynb:134]: <p>Also, the escapes are actually in the equation in the notebook. I think you must have added them some time ago.</p>

**Review by tkphd — APPROVED** [review]
LGTM. Thanks for the hard work @wd15!

**Comment by tkphd** [comment]
On the BM3 spec [Overview](https://random-cat-1485.surge.sh/benchmarks/benchmark3.ipynb/#Overview), what ought to be a paired citation `[17, 18]` renders as `[[17][takaki2014phase], 18]`.

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1485.surge.sh

**Comment by wd15** [comment]
> On the BM3 spec [Overview](https://random-cat-1485.surge.sh/benchmarks/benchmark3.ipynb/#Overview), what ought to be a paired citation `[17, 18]` renders as `[[17][takaki2014phase], 18]`.

[Fixed now](https://random-cat-1485.surge.sh/benchmarks/benchmark3.ipynb/#Overview). Merging.

**Pull request merged**